app/views/layouts/_navigation.html.haml

```diff
@@ -58,15 +58,15 @@
               %span.fa.fa-user
               Sign In
               %span.caret
-            .dropdown-menu{:style => "padding: 17px; min-width: 225px;"}
+            .dropdown-menu
               - if ENV['OSEM_ICHAIN_ENABLED'] == 'true'
                 = form_tag User.ichain_login_url do
                   = text_field_tag 'username', nil, id: 'user_ichain_email_dd', placeholder: 'Username'
                   = password_field_tag 'password', nil, id: 'user_ichain_password_dd', placeholder: 'Password'
                   %button.btn.btn-success.btn-block Sign in
               - else
                 = form_tag new_user_session_path do
-                  = text_field_tag 'user[login]', nil, id: 'user_login_dd', placeholder: 'Username / E-Mail'
+                  = text_field_tag 'user[login]', nil, id: 'user_login_dd',placeholder: 'Username / E-Mail'
```

Comment:
oh, sorry, I missed this detail before.. I don't think this line needs to change